### ISSUE TITLE
xgox core

### ISSUE THREAD

**Issue by gittempo** [exander77/xgox-cli#1]
is there any documentation for installing xgox core into linux and windows server? So i can provide wallet deposit and withdraw service to our users?

**Comment by exander77** [comment]
XGOX deamon is just a single binary program (xgoxd). No installation is needed. You just need to set your server to start the binary (xgoxd) on its startup.

**Comment by gittempo** [comment]
After installing npm we can use this project?
And are we able to create new wallets and send and receive xgox coins from any exchange?

Is there any documentation that can help step by step to understanding the whole things that can be integrated with php

**Comment by exander77** [comment]
Yes, npm and nodejs. This tool basically manages the xgoxd daemon. So you can tell it to create address or send.

**Comment by gittempo** [comment]
Thank you, 
Shall i feel free to ask questions if i need to ask something while testing?

**Comment by gittempo** [comment]
Where i can download xgox daemon?

**Comment by exander77** [comment]
Sure, Linux build is available here: https://github.com/exander77/xgox/releases
